services/QuillLMS/spec/factories/gengo_jobs.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: gengo_jobs
+#
+#  id                 :bigint           not null, primary key
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  english_text_id    :integer          not null
+#  translated_text_id :integer
+#  translation_job_id :string           not null
+#
+FactoryBot.define do
+  factory :gengo_job do
+    english_text_id { 1 }
```

Comment:
I'd like to see this done with an association so that the underlying model gets created in case it matters for a spec:
```
association :english_text
```